languages/common-lisp/concepts/integers/about.md

```diff
@@ -1 +1,8 @@
-TODO: add information on integers concept
+Integers are whole numbers without a decimal point (like `-6`, `0`, `25`, `1234`, etc.)
```

Comment:
In this docuent we probably want to discuss fixnum and bignum. Using those turns since those are in the spec as types. Also mention the limits to these values. Anyone coming from other languages (esp. something like C) may wonder if an integer is 2^32 or 2^64 etc.